t/lifecycle.t

```diff
@@ -0,0 +1,116 @@
+use strict;
+use warnings;
+use 5.14.2;
+
+use Test::More tests => 2;
+use Test::Exception;
+use Test::NoWarnings;
```

Comment:
It seems that PostgreSQL emits warnings when trying to DROP a table that does not exist. This makes the tests in `t/lifecycle.t` fail.

Reply:
Good catch!